pkgs/development/python-modules/pyroaring/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+with python3.pkgs;
+buildPythonPackage rec {
+  pname = "pyroaring";
+  version = "1.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Ezibenroc";
+    repo = "PyRoaringBitMap";
+    rev = version;
+    hash = "sha256-pnANvqyQ5DpG4NWSgWkAkXvSNLO67nfa97nEz3fYf1Y=";
+    fetchSubmodules = true;
+  };
+
+  build-system = [
+    setuptools
+    wheel
+    cython
+  ];
+
+  nativeCheckInputs = [
+    pytest
+    hypothesis
+    unittestCheckHook
+  ];
```

Comment:
```suggestion
  nativeCheckInputs = [
    pytestCheckHook
    hypothesis
  ];
```

Reply:
This is needed, see https://github.com/Ezibenroc/PyRoaringBitMap/blob/master/test.py#L5